### PR TITLE
aide 0.17

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
<details>
<summary>build status</summary>

```
==> make install
/Applications/Xcode.app/Contents/Developer/usr/bin/make  install-am
depbase=`echo src/aide.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
	clang -DHAVE_CONFIG_H -I.  -I. -I./include -I./src -I./src  -D_GNU_SOURCE -W -Wall -g -g -O2  -MT src/aide.o -MD -MP -MF $depbase.Tpo -c -o src/aide.o src/aide.c &&\
	mv -f $depbase.Tpo $depbase.Po
depbase=`echo src/base64.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
	clang -DHAVE_CONFIG_H -I.  -I. -I./include -I./src -I./src  -D_GNU_SOURCE -W -Wall -g -g -O2  -MT src/base64.o -MD -MP -MF $depbase.Tpo -c -o src/base64.o src/base64.c &&\
	mv -f $depbase.Tpo $depbase.Po
depbase=`echo src/be.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
	clang -DHAVE_CONFIG_H -I.  -I. -I./include -I./src -I./src  -D_GNU_SOURCE -W -Wall -g -g -O2  -MT src/be.o -MD -MP -MF $depbase.Tpo -c -o src/be.o src/be.c &&\
	mv -f $depbase.Tpo $depbase.Po
depbase=`echo src/commandconf.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
	clang -DHAVE_CONFIG_H -I.  -I. -I./include -I./src -I./src  -D_GNU_SOURCE -W -Wall -g -g -O2  -MT src/commandconf.o -MD -MP -MF $depbase.Tpo -c -o src/commandconf.o src/commandconf.c &&\
	mv -f $depbase.Tpo $depbase.Po
depbase=`echo src/attributes.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
	clang -DHAVE_CONFIG_H -I.  -I. -I./include -I./src -I./src  -D_GNU_SOURCE -W -Wall -g -g -O2  -MT src/attributes.o -MD -MP -MF $depbase.Tpo -c -o src/attributes.o src/attributes.c &&\
	mv -f $depbase.Tpo $depbase.Po
depbase=`echo src/report.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
	clang -DHAVE_CONFIG_H -I.  -I. -I./include -I./src -I./src  -D_GNU_SOURCE -W -Wall -g -g -O2  -MT src/report.o -MD -MP -MF $depbase.Tpo -c -o src/report.o src/report.c &&\
	mv -f $depbase.Tpo $depbase.Po
In file included from src/attributes.c:25:
./include/util.h:63:7: error: conflicting types for 'strnstr'
char* strnstr(char* haystack,char* needle,int n);
      ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/string.h:161:7: note: previous declaration is here
char    *strnstr(const char *__big, const char *__little, size_t __len);
         ^
1 error generated.
make[1]: *** [src/attributes.o] Error 1
make[1]: *** Waiting for unfinished jobs....
In file included from In file included from src/commandconf.csrc/base64.c::2232:
:
./include/aide.h:In file included from 36./include/aide.h::In file included from src/be.c:3610:10: fatal error: 'error.h' file not found
22:
src/aide.c./include/aide.h: fatal error: 'error.h' file not found
:23:
:./include/aide.h:36:10: fatal error: #include "error.h"
         ^~~~~~~~~
36:10'error.h' file not found
#include "error.h"
         ^~~~~~~~~
#include "error.h"
         ^~~~~~~~~
: fatal error: 'error.h' file not found
#include "error.h"
         ^~~~~~~~~
In file included from src/report.c:23:
./include/aide.h:36:10: fatal error: 'error.h' file not found
#include "error.h"
         ^~~~~~~~~
1 error generated.
make[1]: *** [src/base64.o] Error 1
1 error generated.
1 error generated.
1 error generated.
make[1]: *** [src/be.o] Error 1
make[1]: *** [src/aide.o] Error 1
make[1]: *** [src/commandconf.o] Error 1
1 error generated.
make[1]: *** [src/report.o] Error 1
make: *** [install] Error 2
```

</details>

File: Formula/aide.rb [hunk 1]
```diff
@@ -30,6 +31,11 @@ class Aide < Formula
   uses_from_macos "curl"
 
   def install
+    # fix `fatal error: 'error.h' file not found`
+    inreplace "include/aide.h", "#include \"error.h\"", ""
```
Comment (carlocab) [inline Formula/aide.rb:35]: This probably needs to be reported upstream, no?

Comment (chenrui333) [reply Formula/aide.rb:35]: Yeah, I will submit a patch PR.

Comment (carlocab) [reply Formula/aide.rb:36]: Submitted upstream at https://github.com/aide/aide/pull/85

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**